### DISCUSSION TITLE
Better readable web console

### DISCUSSION THREAD

**Discussion by zaum** [arendst/Tasmota#22592] [Ideas]
Just a small change to make the web console easier to read. Instead of writing "Configure..." on each button, it can appear only once on the screen.  We can save some bytes too :)
![1](https://github.com/user-attachments/assets/88edca6b-554f-478a-b03e-9a6d842ea02f)
![2](https://github.com/user-attachments/assets/aba18ff9-f748-4ed9-a3c2-0f61089eaaf0)
![3](https://github.com/user-attachments/assets/bfe05b27-59ec-4cf5-81a3-7413f3a62a34)

**Comment by arendst** [comment]
Thx. Leaving out "Configure" completely saves even more space.

Have a look how other languages have solved it. Italian is a nice way (no starting configure).

So simply removing "Configure" might be a final solution.

BTW I already hear some rumble in the back of people complaining where "Configure" has gone....

**Comment by arendst** [comment]
Try latest dev with your suggestions.

![image](https://github.com/user-attachments/assets/a091430a-c013-438e-a556-99be9a58c21d)

Now find a nice name for `Auto-configuration` other than `Auto`

**Reply by Wintespe** [reply]
Auto Setup
